CNV/Makefile

```diff
@@ -0,0 +1,8 @@
+.PHONY: all run upgrade
+all: run
```

Comment:
Is there a more descriptive target instead of  "run"?

Did you want to install a default target?